### ISSUE TITLE
warning , Animated:` useNativeDriver` was not specified.

### ISSUE THREAD

**Issue by ilhamsa1** [jacklam718/react-native-modals#228]
hi, i got a warning,
"Animated:` useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or` false` "

I think this issue is caused by the React Native update to version 0.62  [(changelog#deprecated)](https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#deprecated) 

do you have a plan to update this package?

**Comment by nhasoenhasan** [comment]
do you solve this issue??

**Comment by cuongnm2301** [comment]
How can I fix this issue ? Anyone

**Comment by choudharyjji** [comment]
Same issue

**Comment by dchaebae** [comment]
Same issue here.

**Comment by josepgomes** [comment]
Same, it should be a easy fix

**Comment by anilkarwa** [comment]
Same issue not able to use even modal button or any thing, please suggest work around or fix.

**Comment by mangdin** [comment]
remove zhe react-native-popup-dialog.
install zhe new react-native-modals.
i try it ok.

**Comment by hungng147147** [comment]
@mangdin but i never install react-native-popup-dialog?? can you explain more detail

**Comment by blueturtle13g** [comment]
the same issue

**Comment by owlpro** [comment]
same issue | Unprincipled solution 

```
YellowBox.ignoreWarnings([
    'Require cycle: node_modules/',
    'Animated: `useNativeDriver` was not specified',
    'Animated.event now requires a second argument for options'
]);
```

**Comment by NetPumi2** [comment]
I went through this library and `useNativeDriver` is there almost everywhere by default set `false` ... it is weird... and this error has  side-effect -> buttons can not be press for the first time

**Comment by Billwall2020** [comment]
Any news on that one? Still facing the same issue.

**Comment by akinlekan28** [comment]
I am also facing the same issue

**Comment by isipisi281** [comment]
Same here.

**Comment by visionmeksasi** [comment]
Same here.

**Comment by PaitoAnderson** [comment]
Probably the two instances of `Animated.spring` here: https://github.com/jacklam718/react-native-modals/blob/master/src/components/DraggableView.js#L184

**Comment by ccanalesdote** [comment]
any news?

**Comment by quyenuaena** [comment]
any news?

**Comment by tiborelsan** [comment]
up post

**Comment by dntzee** [comment]
any news?

**Comment by jacklam718** [comment]
Sorry for the late response. This issue has been fixed in version `v0.19.10` Thanks @PaitoAnderson's pull request.
